clang/include/clang/Tooling/DependencyScanningTool.h

```diff
@@ -137,8 +137,8 @@ class DependencyScanningTool {
   ///                           arguments for dependencies.
   /// @return An instance of \c TranslationUnitDeps if the scan is successful.
   ///         Otherwise it returns an error.
-  llvm::Expected<dependencies::TranslationUnitDeps>
-  computeDependenciesByNameWithContext(
+  llvm::Expected<clang::dependencies::TranslationUnitDeps>
```

Comment:
`clang::` should be unnecessary here now that the tool is no longer in `clang::tooling::dependencies` but just `clang::tooling`.